packages/components/tests/integration/components/hds/table/th-sort-test.js

```diff
@@ -36,4 +31,14 @@ module('Integration | Component | hds/table/th-sort', function (hooks) {
 
     assert.dom('[data-test-table-th-sort]').hasClass('hds-table__th-sort');
   });
+  test('if @sortOrder is not defined, the swap-vertical icon should be displayed', async function (assert) {
+    await render(hbs`<Hds::Table::ThSort
+  @sortBy='artist'
+  @setSortBy={{this.setSortBy}}
+>artist</Hds::Table::ThSort>`);
+
+    assert
+      .dom(this.element.querySelector('.flight-icon.flight-icon-swap-vertical'))
```

Comment:
I think we can simplify this to to `assert.dom('.flight-icon-swap-vertical').exists();` – there shouldn't be anything else in the DOM as it gets cleared after each test

Reply:
@alex-ju I re-worked the table tests in the PR after this, I would like to review the tests in that PR if that's okay. WDYT?

Reply:
Yes, that makes sense, I've seen the other PR after dropping this comment 😁